website/views/shared/profiles/achievements.jade

```diff
@@ -226,6 +226,18 @@ div(ng-if='::profile.achievements.costumeContest')
     =env.t('costumeContestText')
   hr
 
+div(ng-if='::profile.achievements.greeting')
+  .achievement.achievement-greeting
+  h5=env.t('cheeryChum')
```

Comment:
`cheeryChum` is cute, but we could refactor these into mixins if we renamed them something like `greetingCardAchievementTitle` and `greetingCardAchievementText`

Any reason this is a bad idea?